k-distribution/tests/pyk/Makefile

```diff
@@ -38,7 +37,10 @@ VENV     := venv/pyvenv.cfg
 ACTIVATE := . venv/bin/activate
 
 $(VENV):
-	virtualenv -p python3.8 venv && $(ACTIVATE) && pip install graphviz typing_extensions
+	virtualenv -p python3.8 venv
+
+install-pyk: $(VENV)
+	$(ACTIVATE) && pip install --force-reinstall -e $(K_LIB)/pyk
```

Comment:
Essentially this snippet shows the purpose of the PR.

This is useful for downstream tools that can now install `pyk` with a single command, instead of installing its dependencies and setting up `$PYTHONPATH`.